apps/aecore/lib/aecore/peers/worker.ex

```diff
@@ -219,7 +216,7 @@ defmodule Aecore.Peers.Worker do
                 {:ok, info}
             end
           true ->
-            {:error, :peer_has_own_nonce}
+            {:error, "Equal peer nonces"}
```

Comment:
why is the `:peer_has_own_nonce` removed, if we don't have it we will log an error for everytime we try to add ourself, we should just ignore this,and assume everything is ok

Reply:
I agree we should revert that.

Reply:
please don't merge next time if there are still open comments ;)

Reply:
If you check the timing you will see I merged one day after my approval and before you commented ;) But I agree it was a mistake.